README.md

```diff
@@ -1,2 +1,197 @@
+
+
+Portfolio: finish mobile version is a project where no complex coding is required and the only goal here is to master all of the tools and best practices we learned at Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">index.html</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.html</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- Portfolio: setup and mobile first project is a developer's portfolio project in which you can get Developer Fentahun Agmas's specific social media links and some descriptions.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link (not yet deployed)](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Installation of Git
+installation of NPM
+a working browser
+live server
+Internet connection 
+A code editor like Visual Studio Code are required.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder(your working machine)
+  git clone git@github.com:Tek2022/Finish-mobile_version.git
+
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd my-project
+  To install Webhint
+  npm init -y
+  npm install --save-dev hint@7.x
+  To install Stylelint
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+```
+
+### Usage
+
+To run the project, open the project by launching the Live Server plugging in visual studio code.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Teklu Addise**
+
+- GitHub: [@githubFentahun](https://github.com/Tek2022)
+- Twitter: [@twitterFentahun](https://twitter.com/AddiseTeklu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tekab-addise-897325227/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **[There will be  a form that will be used to contact the developer.]**
+
+- **[Responsive website for different screen sizes like desktop or tablet .]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Tek2022/Finish-mobile_version/issues).
```

Comment:
- [ ] As requesteed by the previouse reviewer kindly Kindly make sure that all your README links are working. This issue link throws a 404 error page.
If you are finding any difficulty, please use the lines below or refer to the [Microverse Readme Template](https://github.com/microverseinc/readme-template).